crates/solidity/testing/sanctuary/src/tests.rs

```diff
@@ -100,21 +104,45 @@ pub fn run_test(file: &SourceFile, events: &Events) -> Result<()> {
 
     let parser = Parser::create(version.clone())?;
     let output = parser.parse(NonterminalKind::SourceUnit, &source);
+    let source_id = file.path.strip_repo_root()?.unwrap_str();
```

Comment:
This always runs bindings for all tests. I thought you wanted to make it optional for now? did that change?